src/Symfony/Bundle/SecurityBundle/Resources/config/security_authenticator_access_token.php

```diff
@@ -26,9 +26,9 @@
         ->set('security.authenticator.access_token', AccessTokenAuthenticator::class)
             ->abstract()
             ->args([
-                abstract_arg('user provider'),
                 abstract_arg('access token handler'),
                 abstract_arg('access token extractor'),
+                abstract_arg('user provider'),
```

Comment:
Are `abstract_arg` required to be configured and should this become null instead?

```suggestion
               null,
```

Reply:
Yes